cmd/kops/get_cluster.go

```diff
@@ -150,12 +150,20 @@ func RunGetClusters(context Factory, out io.Writer, options *GetClusterOptions)
 	case OutputTable:
 		return clusterOutputTable(clusters, out)
 	case OutputYaml:
-		return clusterOutputYAML(clusters, out)
+		var obj []runtime.Object
+		for _, c := range clusters {
+			obj = append(obj, c)
+		}
+		return fullOutputYAML(out, obj...)
 	case OutputJSON:
-		return clusterOutputJson(clusters, out)
+		var obj []runtime.Object
+		for _, c := range clusters {
+			obj = append(obj, c)
+		}
+		return fullOutputJSON(out, obj...)
 
 	default:
-		return fmt.Errorf("Unknown output format: %q", options.output)
+		return fmt.Errorf("unknown output format: %q", options.output)
```

Comment:
So the reason for these being upper case is because they look better when displayed, and this is a CLI command object.  It's the same way the go rules conflict with all our comments in apimachinery types.

I guess we should format the error forcing the first character to upper-case?  But in another PR.  Please no more case-switches in this PR though.

Reply:
I will back them out, they bug me